src/hooks/use-update-user.tsx

```diff
@@ -18,6 +21,7 @@ const useUpdateUser = (userId: string) => {
       severity: snackbarVariants.success,
       message: 'editProfilePage.profile.successMessage'
     })
+    navigate(0)
```

Comment:
where it will navigate?

Reply:
It will reload the page after successful photo update.
It was advice of the expert to use router for reload